src/test/java/wolox/training/controllers/UserControllerTest.java

```diff
@@ -216,6 +223,30 @@ void whenRemoveBookThenReturnStatusNoContent() throws Exception {
                 .andExpect(status().isNoContent());
     }
 
+    @WithMockUser(value = "miguel")
+    @Test
+    @DisplayName("test, When find authenticated user , it return status OK")
+    void whenFindAuthenticatedUserThenRetunrStatusOK() throws Exception {
```

Comment:
You should test it when not found user

Reply:
when the user is not authenticated, the security component responds with a 401 Unauthorized